server/core/src/test/java/org/infinispan/server/core/DummyServerManagement.java

```diff
@@ -1,23 +1,43 @@
 package org.infinispan.server.core;
 
 import java.nio.file.Path;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.CompletionStage;
 
 import javax.sql.DataSource;
 
 import org.infinispan.commons.configuration.io.ConfigurationWriter;
 import org.infinispan.lifecycle.ComponentStatus;
 import org.infinispan.manager.DefaultCacheManager;
+import org.infinispan.manager.EmbeddedCacheManager;
 import org.infinispan.tasks.TaskManager;
+import org.mockito.Mockito;
 
 /**
  * @author Tristan Tarrant &lt;tristan@infinispan.org&gt;
  * @since 13.0
  **/
 public class DummyServerManagement implements ServerManagement {
 
+   private DefaultCacheManager defaultCacheManager;
+   private Map<String, ProtocolServer> protocolServers;
+   private ServerStateManager serverStateManager;
+
+   public DummyServerManagement() {
```

Comment:
I feel like the name no longer applies 😞

Reply:
yes, we can move it to mock instead of dummy